source/php-libraries.txt

```diff
@@ -68,6 +68,10 @@ Framework Integrations
 
 - Symfony
 
+  - :ref:`php-symfony-integration` describes the advantages of using MongoDB
```

Comment:
FWIW, I didn't pick up on any "advantages" in the tutorial. It seems more informative than marketing spiel, which I'd consider a good thing.